### DISCUSSION TITLE
Can i have 2 or more Postal in the same domain ?

### DISCUSSION THREAD

**Discussion by CryptosWORLD** [postalserver/postal#1864] [Help with using Postal]
hi , i m interesting how to setup 2 **Postal in the same domain** ( no ip rotation ) ?
one is in US and the other is in EU may be in the future one in Japan or Australia 
how will be their config to load balance even be backup the one to the other ?
how will be the MX record same priority an both ?
will be needed a load balancer like haproxy ?

**Comment by willpower232** [comment]
If both installations of Postal have access to the same database and rabbitmq then it doesn't matter where the various parts of Postal are hosted.

I'd imagine you can use any load balancer or reverse proxy to handle traffic. You would have to look at what is available from your hosting provider.
